lib/note-info/index.tsx

```diff
@@ -19,9 +22,15 @@ type StateProps = {
   note: T.NoteEntity | null;
 };
 
-type Props = StateProps;
+type DispatchProps = {
+  onToggleNoteInfo: () => any;
```

Comment:
We've added this type but then our contract breaks in `mapDispatchToProps` (and I'm seeing that error in my IDE) because the component as written expects and is provided `onOutsideClick`

We need to _either_ rename this here to `onOutsideClick` _or_ rename all of the `onOutsideClick` references to `onToggleNoteInfo`